mithril-common/src/test_utils/apispec.rs

```diff
@@ -660,4 +768,174 @@ mod tests {
         assert!(!spec_files.is_empty());
         assert!(spec_files.contains(&APISpec::get_default_spec_file()))
     }
+
+    #[test]
+    fn test_examples_conformity() {
```

Comment:
Where should we put this test that validate the openapi.yaml file of the project while the other tests validate the tooling.

Reply:
I would move it to common `lib.rs`, but since apispec is behind a feature there's no real "fitting" place for it.

Something like this patch:
```diff
diff --git a/mithril-common/src/lib.rs b/mithril-common/src/lib.rs
index b0c49f43a7..363fcffc75 100644
--- a/mithril-common/src/lib.rs
+++ b/mithril-common/src/lib.rs
@@ -88,3 +88,21 @@ pub const MITHRIL_API_VERSION_HEADER: &str = "mithril-api-version";
 
 /// Mithril Signer node version header name
 pub const MITHRIL_SIGNER_VERSION_HEADER: &str = "signer-node-version";
+
+#[cfg(test)]
+mod tests {
+    #[cfg(feature = "apispec")]
+    #[test]
+    fn test_openapi_examples_conformity() {
+        use crate::test_utils::apispec::APISpec;
+        let api_spec = APISpec::from_file(&APISpec::get_default_spec_file());
+
+        let errors: Vec<String> = api_spec.verify_examples();
+
+        assert!(
+            errors.is_empty(),
+            "Errors in examples\n{}",
+            errors.join("\n")
+        );
+    }
+}

```